### DISCUSSION TITLE
handbook/enterprise_security/devsecops/metrics/

### DISCUSSION THREAD

**Discussion by giscus** [SEC-CAFE/handbook#8] [General]
# handbook/enterprise_security/devsecops/metrics/

企业软件研发安全成熟度度量标准与指标

https://sec.cafe/handbook/enterprise_security/devsecops/metrics/

**Comment by user1430** [comment]
选目标比做目标更重要，选错就是做错，越做越错。慎重回应老板提出的度量建设活动

**Reply by fooying** [reply]
嗯，选择挺重要的，可以考虑先选择可达和关键指标，再慢慢迭代版本扩展其他指标。
度量建设还是挺重要的，因为跟老板去说什么今天检测、修复多少漏洞，明天又多少其实不管数字大小，老板角度会没什么感觉；因为老板对于安全这东西，不管是合规还是实际安全角度，都觉得挺重要的，但该怎么评估安全做的好，他也是没概念的，所以又个度量指标体系，其实一方面可以回答这个问题，也帮助安全团队评估现状和呈现价值，同时用的好，也是OKR制定、人力扩充等比较好一种借力。
